sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/pipeline_job.py

```diff
@@ -345,19 +345,30 @@ def _try_get_data_binding(_input_output_data) -> Union[str, None]:
                     return None
                 return _input_output_data._data_binding()
 
+            def _is_group_input(_input_obj) -> bool:
```

Comment:
How about GroupInput._is_group_attr_dict?

Reply:
Great, have replaced with this.